maistra/common.sh

```diff
@@ -33,4 +33,8 @@ if [ -n "${BAZEL_REMOTE_CACHE}" ]; then
   COMMON_FLAGS+=" --remote_cache=${BAZEL_REMOTE_CACHE} "
 elif [ -n "${BAZEL_DISK_CACHE}" ]; then
   COMMON_FLAGS+=" --disk_cache=${BAZEL_DISK_CACHE} "
-fi
\ No newline at end of file
+fi
+
+if [ -n "${BAZEL_EXPERIMENTAL_REMOTE_DOWNLOADER}" ]; then
```

Comment:
Hey @tedjpoole out of curiosity, what's the difference between this flag and `--remote_cache` as specified here: https://github.com/buchgr/bazel-remote#configuring-bazel ?

Reply:
`--experimental_remote_downloader` applies to downloads, whereas the `--remote_cache` option applies to build actions. For me, with a relatively slow internet connection, this speeds up downloading all the third party packages.